docs/source/notes/quickstart.md

```diff
@@ -16,166 +14,51 @@ to train other models in MMF.
 
 If you use MMF in your work, please cite:
 
-```text
-@inproceedings{Singh2019TowardsVM,
-  title={Towards VQA Models That Can Read},
-  author={Singh, Amanpreet and Natarajan, Vivek and Shah, Meet and Jiang, Yu and Chen, Xinlei and Batra, Dhruv and Parikh, Devi and Rohrbach, Marcus},
-  booktitle={Proceedings of the IEEE Conference on Computer Vision and Pattern Recognition},
-  year={2019}
-}
-```
-
-and
-
-```text
-@inproceedings{singh2019pythia,
-  title={Pythia-a platform for vision \& language research},
-  author={Singh, Amanpreet and Natarajan, Vivek and Jiang, Yu and Chen, Xinlei and Shah, Meet and Rohrbach, Marcus and Batra, Dhruv and Parikh, Devi},
-  booktitle={SysML Workshop, NeurIPS},
-  volume={2018},
-  year={2019}
-}
-```
+[Add new citation]
 
 ## Installation
 
-1. Clone MMF repository
+Install MMF using pip
 
-```bash
-git clone https://github.com/facebookresearch/mmf ~/mmf
 ```
-
-2. Install dependencies and setup
-
-```bash
-cd ~/mmf
-python setup.py develop
+pip install --upgrade --pre mmf
```

Comment:
Link to installation docs and mention installing from source. I would directly link the installation docs here instead of mentioning anything.